src/shared/components/Spinner/Spinner.styles.ts

```diff
@@ -1,12 +1,13 @@
 import { styled } from '@mui/system';
 
-import { StyledFlexAllCenter } from 'shared/styles/styledComponents';
-import theme from 'shared/styles/theme';
+import { theme, StyledFlexAllCenter } from 'shared/styles';
+import { shouldForwardProp } from 'shared/utils';
 
-export const StyledSpinner = styled(StyledFlexAllCenter)`
+export const StyledSpinner = styled(StyledFlexAllCenter, shouldForwardProp)`
   position: absolute;
   width: 100%;
   height: 100%;
-  background-color: rgb(255 255 255 / 50%);
+  background-color: ${({ noBackground }: { noBackground?: boolean }) =>
+    noBackground ? 'transparent' : 'rgb(255 255 255 50%)'};
```

Comment:
'rgb(255 255 255 50%)' - in this format the color is displayed? no need for a slash 'rgb(255 255 255 / 50%)' ?
and maybe it makes sense to move it to variables?